jupyterlab_ucaip/jupyterlab_ucaip/service.py

```diff
@@ -404,11 +404,23 @@ def get_endpoints(self, model_id):
         endpoints.append(built)
     return endpoints
 
-  def check_deploying(self, model_name):
+  def get_all_endpoints(self):
+    gcp_endpoints = self._endpoint_client.list_endpoints(parent=self._parent).endpoints
+    endpoints = []
+    for endpoint in gcp_endpoints:
+      built = self._build_endpoint("None", endpoint)
+      endpoints.append(built)
+    return endpoints
+
+  def check_deploying(self, model_name, endpoint_id):
```

Comment:
It's not clear to me on its face what this method is doing. check_deploying seems like it would return a boolean, not a list. Can we rename or clarify with comments?

Reply:
A rename would be good here you're right